tests/ui/image/implicit_not_in_fragment.stderr

```diff
@@ -0,0 +1,18 @@
+error: ImageSampleImplicitLod cannot be used outside a fragment shader
+  |
+  = note: Stack:
+          <spirv_std::image::Image<f32, 1, 2, 0, 0, 1, 0>>::sample::<f32, glam::vec4::Vec4, glam::vec2::Vec2>
+          implicit_not_in_fragment::deeper_stack
+          implicit_not_in_fragment::deep_stack
+          implicit_not_in_fragment::main
+          Unnamed function ID %79
```

Comment:
It would be nice to make `get_name` give something like `entry "main"` for the entry-point, to alleviate any potential confusion (in both this and zombie errors).